src/core/layout/gel-defaults.js

```diff
@@ -20,6 +20,8 @@ const addChannelToAll = (defaults, channel) => Object.fromEntries(Object.entries
 
 export const buttonsChannel = screen => (screen ? `gel-buttons-${screen.scene.key}` : "gel-buttons");
 export const config = screen => {
+	const { verticalTopRight } = screen?.cache.json.get("game-config") || {};
```

Comment:
As this is not just for the game component, but is more global, could we call this `config.json` ? It's already at top level in the theme folder so this should make sense.